test/lint/lint-circular-dependencies.sh

```diff
@@ -21,6 +21,9 @@ EXPECTED_CIRCULAR_DEPENDENCIES=(
     "wallet/fees -> wallet/wallet -> wallet/fees"
     "wallet/wallet -> wallet/walletdb -> wallet/wallet"
     "policy/fees -> txmempool -> validation -> policy/fees"
+    "auxpow -> primitives/block -> auxpow"
+    "dogecoin -> pow -> dogecoin"
+    "rpc/blockchain -> rpc/rawtransaction -> rpc/blockchain"
```

Comment:
Out of curiosity, I don't see any of these being introduced with this PR, wondering if this is sitting in the right commit?